docs/topics/native/native-objc-interop.md

```diff
@@ -32,7 +32,7 @@ The table below shows how Kotlin concepts are mapped to Swift/Objective-C and vi
 | `suspend` -> | `completionHandler:`/ `async` | `completionHandler:` | [note](#errors-and-exceptions) [note](#suspending-functions) |
 | `@Throws fun` | `throws` | `error:(NSError**)error` | [note](#errors-and-exceptions) |
 | Extension | Extension | Category member | [note](#extensions-and-category-members) |
-| `companion` member <- | Class method or property | Class method or property |  |
+| `companion` member <- | `shared` or `companion` property | `shared` or `companion` property | [note](#kotlin-singletons)  |
```

Comment:
This row of the table described how Objective-C class methods and properties are imported to Kotlin -- as companion members. It didn't change, and the new version is incorrect. Additionally, the link to `#kotlin-singletons` is unrelated.

So please update the row below instead:
```
| `Singleton` | `Singleton()` | `[Singleton singleton]` | [note](#kotlin-singletons) |
```